engines/bops_config/app/controllers/bops_config/application_controller.rb

```diff
@@ -2,6 +2,16 @@
 
 module BopsConfig
   class ApplicationController < ActionController::Base
+    include BopsCore::AuditableController
+
+    self.audit_payload = -> {
+      {
+        engine: "bops_config",
+        params: request.path_parameters,
+        user: current_user.audit_attributes
+      }
+    }
```

Comment:
This is a common payload that's added to all audit events - it's executed in the context of the controller so has access to instance variables and methods.